source/JABHandler.py

```diff
@@ -775,9 +777,5 @@ def terminate():
 	bridgeDll.setPropertyCaretChangeFP(None)
 	h=bridgeDll._handle
 	bridgeDll=None
-	if legacyAccessBridge:
-		del cdll.windowsAccessBridge 
-	else:
-		delattr(cdll,'windowsAccessBridge-32')

```

Comment:
Should this stay?

Reply:
No. When you use cdll.LoadLibrary (which we now do so we can specify the full path), the ctypes loader cache isn't used:

> __getattr__() has special behavior: It allows loading a shared library by accessing it as attribute of a library loader instance. The result is cached, so repeated attribute accesses return the same library each time.
> LoadLibrary(name)
> Load a shared library into the process and return it. This method always returns a new instance of the library.

https://docs.python.org/3/library/ctypes.html#ctypes.LibraryLoader